src/AddTodoForm.js

```diff
@@ -0,0 +1,24 @@
+import React from "react"
+
+export default function AddTodoForm (props) {
+
+    //takes submit even as argument
+    //sets value of input as variable and then sets state to value of input variable
+    let handleAddTodo = (event) => {
+        event.preventDefault();
+        let todoTitle = event.target.elements.title.value;
+        console.log(todoTitle);
+        props.onAddTodo(prevTodo => ({...prevTodo, value: todoTitle}));
```

Comment:
```suggestion
        props.onAddTodo(todoTitle);
```

I think you can pass the value straight into `onAddTodo`. It looks like you added an object as your useState variable. This may mess you up in later lessons because we will add this item to our todo array soon.